front/src/applications/operationalStudies/views/Scenario/components/SimulationResults/SimulationResults.tsx

```diff
@@ -142,6 +142,9 @@ const SimulationResults = ({
       (timetableItem) => timetableItem.id === extractPacedTrainIdFromOccurrenceId(selectedTrainId)
     );
     if (!pacedTrain || !isPacedTrainWithDetails(pacedTrain)) return undefined;
+
+    if (!pacedTrain.paced) return pacedTrain.summary;
```

Comment:
This is never supposed to happen because `selectedTrainId` is an occurrence ID, I think?

In that case, we should probably throw an error here?